pom.xml

```diff
@@ -1,65 +1,156 @@
+        <mapstruct.version>1.5.5.Final</mapstruct.version>
+        <retrofit.version>2.9.0</retrofit.version>
         <!-- test dependencies -->
+        <assertj.bom.version>3.24.2</assertj.bom.version>
         <approvaltests.version>22.3.3</approvaltests.version>
-        <instancio-junit.version>4.3.1</instancio-junit.version>
+        <instancio.junit.version>4.3.1</instancio.junit.version>
+        <!-- TODO: not sure if we will require it for testing? -->
         <jimfs.version>1.3.0</jimfs.version>
-        <!-- reporting dependencies-->
-        <jacoco.version>0.8.11</jacoco.version>
+
+        <!-- plugin version-->
+        <spotless-plugin.version>2.40.0</spotless-plugin.version>
+        <enforcer-plugin.version>3.2.1</enforcer-plugin.version>
+        <jsonschema2pojo-plugin.version>1.2.1</jsonschema2pojo-plugin.version>
+        <build-helper-plugin.version>3.4.0</build-helper-plugin.version>
+        <compiler-plugin.version>3.12.1</compiler-plugin.version>
+        <surefire-plugin.version>3.2.5</surefire-plugin.version>
+        <failsafe-plugin.version>3.2.5</failsafe-plugin.version>
+        <jacoco-plugin.version>0.8.11</jacoco-plugin.version>
     </properties>
 
-    <modules>
-        <module>model</module>
-        <module>application</module>
-    </modules>
+    <dependencyManagement>
+        <dependencies>
+            <!-- quarkus -->
+            <dependency>
+                <groupId>${quarkus.platform.group-id}</groupId>
+                <artifactId>${quarkus.platform.artifact-id}</artifactId>
+                <version>${quarkus.platform.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <!-- other-->
+            <dependency>
+                <groupId>com.squareup.okhttp3</groupId>
+                <artifactId>okhttp-bom</artifactId>
+                <version>${okhttp.bom.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+            <dependency>
+                <groupId>io.github.resilience4j</groupId>
+                <artifactId>resilience4j-bom</artifactId>
+                <version>${resilience4j.bom.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+            <!-- testing -->
+            <dependency>
+                <groupId>org.assertj</groupId>
+                <artifactId>assertj-bom</artifactId>
+                <version>${assertj.bom.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
 
-    <!-- common dependencies for all projects -->
     <dependencies>
-        <!-- logging interface should be the same -->
+        <!-- TODO: dependencies to remove -->
+        <!-- TODO: slf4j should be replaced by other logger -->

```

Comment:
Actually I would prefer to keep the slf4j as a portable API, and keep the quarkus adapter implementation. This will make the libraries portable in case that we extract them. Removing the comment.